addons/resource/migrations/12.0.1.1/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,14 @@
+---Fields in module 'resource'---
+resource     / resource.calendar        / hours_per_day (float)         : NEW 
+resource     / resource.calendar.attendance / day_period (selection)        : NEW required: required, selection_keys: ['afternoon', 'morning'], req_default: function
+resource     / resource.calendar.leaves / time_type (selection)         : NEW selection_keys: ['leave', 'other']
```

Comment:
I don't have this clear if the default value `leave` is the proper one for getting the same result as in v11. Have you investigated this?

Reply:
It should be handled in hr_holidays module? The resource.calendar.leaves if linked to hr.holiday with holiday_id, then the resource.calendar.leaves should have the same time_type as its holiday.
See https://github.com/odoo/odoo/blob/12.0/addons/hr_holidays/models/hr_leave.py#L544

Reply:
OK, then we reassign in that module.